Edits to super-sql/queries doc

philrz opened the pull request:
Like #6396, this is a bite-sized edit to one of the docs created/modified in #6363.

In this pass I've started trying to make use of the recent mdtest Playground enhancements from @nwt to drop labels and make them narrower. If the approach here passes scrutiny I'll apply it in similar upcoming edits.

In addition to the inline comments, a couple questions for @mccanne about parts of this doc I spotted but did not propose edits:

1. Is detail about the availability of `|>` planned as part of the SQL docs you're currently drafting? It's mentioned here in passing and if we expect to have text about it I'll make a note-to-self to circle back and link to it. No doubt users that have seen Google's pipe SQL will be familiar with it, but I'm more thinking of users coming to SuperSQL for their first pipe SQL experience being curious when they see it mentioned in passing.

2. Are we standardizing on "SQL Clauses" over "SQL Operators", or are these two different concepts we expect to describe separately in the upcoming SQL docs? I ask because this doc currently links the text "SQL operator" to a doc titled "SQL Clauses" in the `SUMMARY.md`, so I wasn't sure if this was something to make consistent now or if it's about to all become clearer in what you're drafting.

mccanne commented on book/src/super-sql/queries.md:
```diff
@@ -32,15 +32,14 @@ along with an operator sequence in the parentheses having the structure:
-of other scopes defined within the scope.
+[lexical scope](https://en.wikipedia.org/wiki/Scope_(computer_science)#Lexical_scope).
+Bindings created by declarations within the scope are reachable within that
+scope and any nested scopes, but not outside it.
```
This isn't right and needs to be further rephrased

mccanne replied:
sounds good

philrz replied:
Ok, that sounds like consensus. I just pushed a commit that adjusts this to include @nwt's suggestion. I also dropped the "but not outside it" part. I hadn't heard any explicit 👍 or 👎 about that part of my first proposal, but in an offline exchange @nwt confirmed my suspicion that it read as superfluous, so figured I might as well drop it since it wasn't in the original.

philrz commented on book/src/super-sql/queries.md:
````diff
@@ -51,22 +50,34 @@ of references to data input is defined by
-For example, [this example](declarations/constants.md#examples) of a constant declaration
-```
+For example, this query containing a constant declaration emits the value `3.14`
+```mdtest-spq {data-layout='no-labels'} {style='text-align:left;width:20%'}
````
As covered on Zoom this morning, a `make` in the `book/` directory was just needed to see the latest changes including the dropped input pane, so I'm marking this comment Resolved.

philrz commented on book/src/super-sql/queries.md:
```diff
@@ -168,12 +178,11 @@ A text entity represents a string where quotes can be omitted for
 Specifically, a text entity is one of:
-* a string literal (double quoted, single quoted, or raw string),
+* a [string literal](types/string.md) (double quoted, single quoted, or raw string),
 * a path consisting of a sequence of characters consisting of letters, digits, `_`,  `$`,  `.`, and `/`, or
```
@mccanne: Ok, just pushed a commit so now that bullet reads as:

> * an unquoted string consisting of a sequence of characters consisting of letters, digits, `_`,  `$`,  `.`, and `/`, or

philrz commented:
@mccanne: I think I've incorporated the suggested changes to the content, and I've also updated the conventions in the README to reflect your latest preferences regarding labels/width on the inline playground examples. While we've been using the examples here as the reference for the iterations on vertical space and Copy button, since I expect those to come in a separate branch from @nwt maybe you're game to Approve the enhancements here thus far and we'll keep tracking the vertical/Copy as a separate effort.
